src/signals/incident-management/containers/ReporterContainer/types.ts

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2021 Gemeente Amsterdam
+export type Feedback = {
+  is_satisfied: boolean | null
+  submitted_at: string | null
+} | null
```

Comment:
it would be nicer not to have `null` be part of the `Feedback` type.

That way when you see another interface that implements this type you can easily see if it is nullable/optional, and it also allows you to guard for it and pass it down as non nullable to children components etc.